RFCs/Template.md

```diff
@@ -0,0 +1,34 @@
+# RFC-#: Title
+
+**To be Reviewed By:** 
+
+**Authors:**
+
+**Status:** **Active** | Accepted | Dropped | Superseded
+
+**Superseded by:** N/A
+
+**Related:** N/A
+
+
+## Problem
+*What is this proposal solving? Why does the problem matter? How does the problem affect the user?*
+
+### Anti-Goals
+*What is outside the scope of what the proposal is trying to solve?*
+
+## Solution
+*Describe your solution and how it’s going to solve the problem. This is likely the largest section of your proposal and might even include some high-level diagrams if you are proposing code changes. While all important aspects need to be covered, also keep in mind that shorter documents are more likely to be read.*
+
+### User Impact
+*If you are proposing to add or modify public interfaces, those changes should be outlined here in detail.*
+
+### Backwards Compatibility and Upgrade Path
+*Will the regular rolling upgrade process work with these changes?*
+
+*How do the proposed changes impact backwards-compatibility? Are message or file formats changing?*
```

Comment:
not sure about "file formats" in our case, but we should probably include APIs and CLI commands here

Reply:
removed "file formats" from Geode. Added APIs and CLI.